assets/scripts/app/__tests__/NoConnectionMessage.test.js

```diff
@@ -1,44 +1,51 @@
 /* eslint-env jest */
 import React from 'react'
-import { shallow } from 'enzyme'
-import { mountWithIntl as mount } from '../../../../test/helpers/intl-enzyme-test-helper.js'
-import { NoConnectionMessage } from '../NoConnectionMessage'
+import { fireEvent } from '@testing-library/react'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
+import NoConnectionMessage from '../NoConnectionMessage'
 import { nonblockingAjaxTryAgain } from '../../util/fetch_nonblocking'
+import { showNoConnectionMessage } from '../../store/actions/status'
 
 jest.mock('../../util/fetch_nonblocking', () => ({
   nonblockingAjaxTryAgain: jest.fn()
 }))
 
-describe('NoConnectionMessage', () => {
-  it('renders a button', () => {
-    const wrapper = shallow(<NoConnectionMessage />)
-    expect(wrapper.find('button').length).toEqual(1)
-  })
+const initialState = {
+  status: {
+    noConnectionMessage: false
+  }
+}
 
+describe('NoConnectionMessage', () => {
```

Comment:
In the old version of our test, we had a test to see if the button in the component renders. This is covered by other tests later on, so instead of porting this test, we can take the opportunity to remove it.